source/kernel/src/forth.rs

```diff
@@ -435,6 +438,21 @@ async fn spawn_forth_task(forth: &mut forth3::Forth<MnemosContext>) -> Result<()
     Ok(())
 }
 
+/// Binding for [`Kernel::sleep()`]
+///
+/// Sleep for the provided number of milliseconds.
+///
+/// Call: `MILLIS sleep`.
+/// Return: No change
+async fn sleep_ms(forth: &mut forth3::Forth<MnemosContext>) -> Result<(), forth3::Error> {
+    let millis = forth.data_stack.try_pop()?.as_i32();
```

Comment:
You probably want to check this is not negative, otherwise we're gunna be sleeping for a hot minute

Reply:
good call